MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -433,10 +501,25 @@ - (void)acquireTokenInteractive:(id)sender
                 [self updateResultViewError:error];
             }
             
+            [_webView loadHTMLString:@"<html><head></head><body>done!</body></html>" baseURL:nil];
+            [_authView setHidden:YES];
+            [self.view setNeedsDisplay];
+            
             [[NSNotificationCenter defaultCenter] postNotificationName:MSALTestAppCacheChangeNotification object:self];
         });
     };
     
+    application.webviewType = _webviewSelection.selectedSegmentIndex == 0 ? MSALWebviewTypeWKWebView : MSALWebviewTypeAutomatic;
+    
+    if (application.webviewType == MSALWebviewTypeWKWebView &&
+        [[_customWebViewSelection titleForSegmentAtIndex:_customWebViewSelection.selectedSegmentIndex] isEqualToString:TEST_EMBEDDED_CUSTOM])
```

Comment:
nit: can we check _customWebViewSelection selectedSegmentIndex instead?

Reply:
Sure, done